app/exerlog/lib/src/core/base/shared_preference/shared_pref_wrapper.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:convert';
+
+import 'package:exerlog/src/core/base/extensions/string_extension.dart';
+import 'package:exerlog/src/utils/logger/logger.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+late SharedPreferences sharedPreferences;
```

Comment:
wouldn't it be better to wrap all the methods of the shared pref wrapper in class with static methods and not have the sharedPreferences be globally declared? This way we are sure that all interactions with the shared preferences have to go trough this wrapper and people can't just call it directly.